app/pages/sections/order-items/associated-services/dashboard/controller.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */

```

Comment:
what's this one for?

Reply:
Yeah they shouldn't be in. Will delete them